.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+.*node_modules/flow-coverage-report.*
```

Comment:
wow I wouldn't have though of this one 👍

Reply:
it is the one that has type errors when upgrading flow... go figure. I hope it'll be fixed soon so I can de-ignore it. Or maybe ignore all of `node_modules` (let's see what @julienw experiment will say ;)